js/intro.js

```diff
@@ -313,13 +301,17 @@ async function showSavedDetails() {
     });
 
     const selfDetails = await res.json();
-    const userId = getUserId();
     if (selfDetails.roles.super_user) {
       notAuthorized.classList.add('hidden');
       mainContainer.classList.remove('hidden');
-      showSavedDetails();
+      if (!urlParams.has('id')) {
+        loading.classList.add('hidden');
+        generateNoDataFoundPage();
+      } else {
+        showSavedDetails();
+      }
     } else {
-      if (userId == 'wrong url') {
+      if (!urlParams.has('id')) {
```

Comment:
Aren't the conditions on line **307** and **314** are same?

Reply:
+1

Reply:
I was thinking that if normal user go to 'http://localhost:5000/intro.html`
he will get 404 page like superuser get

but if i removed this then normal user always get 401 page